autoextract/__main__.py

```diff
@@ -116,6 +118,12 @@ def read_input(input_fp, intype, page_type):
                    choices=["DEBUG", "INFO", "WARNING", "ERROR"],
                    help="log level")
     p.add_argument("--shuffle", help="Shuffle input URLs", action="store_true")
```

Comment:
@kmike @ivanprado 

I've added these two flags to our command-line utility:

- `--disable-retries`
- `--skip-query-errors`

Since they're flags, they're both false by default, which means:

- `handle_retries=True`
- `skip_query_errors=False`

The last argument has no effect if retries are already disabled. This is documented in the help text and in the request function's docstring.

Let me know your thoughts.

Reply:
I have noticed that we are enabling retries by default. Should we switch this behavior and replace the first flag by something like `--enable-retries` or simply `--retry`? Should we make this change also in the source code or should we keep the current behavior there?